src/component/mxgraph/GraphConfigurator.ts

```diff
@@ -82,18 +82,14 @@ export default class GraphConfigurator {
       // Disable gesture support for zoom
       panningHandler.setPinchEnabled(false);
       // Disable panning on touch device
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      panningHandler.isForcePanningEvent = (me: mxMouseEvent): boolean => false;
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars -- prefix parameter name - common practice to acknowledge the fact that some parameter is unused (e.g. in TypeScript compiler)
+      panningHandler.isForcePanningEvent = (_me: mxMouseEvent): boolean => false;
     }
   }
 
-  private getPanningHandler(cursor: 'grab' | 'default'): OmitThisParameter<(this: BpmnGraph) => void> {
-    return this.getPanningHandlerCallback(cursor).bind(this.graph);
-  }
-
-  private getPanningHandlerCallback(cursor: 'grab' | 'default'): () => void {
-    return function (this: BpmnGraph): void {
-      this.isEnabled() && (this.container.style.cursor = cursor);
+  private getPanningHandler(cursor: 'grab' | 'default'): () => void {
```

Comment:
ℹ️ I have tested this code simpification and the cursor is still correctly updated. Tested on Chrome and Firefox on Ubuntu 20.04.
Please test this again during the review.

Reply:
Tested on Firefox, Chrome & Safari on macOS 12.3.1